bin/ci/travis/run.sh

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
 
 cd ruby-gem
-bundle update
 
 touch lib/calabash-android/lib/TestServer.apk
+touch lib/calabash-android/lib/AndroidManifest.xml
```

Comment:
I think we might want to replace these files with the *actual* contents at some point